tools/validate-mkcert.sh

```diff
@@ -0,0 +1,9 @@
+# shellcheck shell=bash
+if ! command -v mkcert &> /dev/null
+then
+    echo "mkcert is not installed, consider following this instructions: https://github.com/FiloSottile/mkcert#installation "
+    exit
```

Comment:
`exit 1` to indicate error, and make it fail if we use in a pipeline (like any other tasks that has `tls-files` as a dependency)

Reply:
This is not supposed to run as part of a pipeline. But I would like to start using this patter as this could end up in a pipeline at some point.